behave_django/management/commands/behave.py

```diff
@@ -69,6 +72,18 @@ def add_behave_arguments(parser):  # noqa
         '--simple',
     ]
 
+    def valid_python_module(path):
+        try:
+            return import_string(path)
+        except ImportError:
+            raise ArgumentTypeError("No module named '%s' was found." % path)
+
+    parser.add_argument(
+        '--runner', action='store', dest='runner_class',
+        default='behave.runner.Runner', type=valid_python_module,
+        help='Tells Behave to use a specific runner. (default: %(default)s)',
```

Comment:
This is actually the _Behave_ runner, not any of [those _we_ provide](https://github.com/behave/behave-django/blob/master/behave_django/runner.py) in this project. I'm fascinated!

Reply:
While I start liking this PR more and more I now start <s>worrying that </s>wondering whether :smirk: this feature should actually go into Behave. What do you think about that? :thinking:

Reply:
I'm looking forward to your feedback here. The *behave* command actually [has no `-r --runner` option](https://behave.readthedocs.io/en/latest/behave.html), currently. Maybe it would be desirable to have that possibility also for plain *Behave*, not only tied to using it with Django, hmm?

I would still merge your efforts here, probably, but:

* name the option appropriately `--behave-runner` (as you had it before) *and*
* open a new issue to discuss migrating the feature to the *Behave* code base.

I hope that makes sense.